recipe/meta.yaml

```diff
@@ -1,16 +1,14 @@
 {% set name = "coveralls" %}
-{% set version = "1.1" %}
-{% set hash_type = "sha256" %}
-{% set hash = "34160385c13b0c43691ab11c080d4b10dabe3280fc0b2173c731efc5db836808" %}
+{% set version = "1.2.0" %}
+{% set git_url = "https://github.com/coveralls-clients/coveralls-python" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash }}
```

Comment:
This `hash_type/hash` jinja2 is silly. You can modify that to be just `sha256: 510682001517bcca1def9f6252df6ce730fcb9831c62d9fff7c7d55b6fdabdf3` (that is already the sha256 for version 1.2.0.)